packages/harmony/src/components/input/TextInput/HelperText.tsx

```diff
@@ -0,0 +1,27 @@
+import type { ReactNode } from 'react'
+
+import { Text, TextSize } from 'components/typography'
+
+import styles from './HelperText.module.css'
+
+type HelperTextProps = {
+  children: ReactNode
+  hasError?: boolean
+  size?: TextSize
+}
+
+export const HelperText = (props: HelperTextProps) => {
+  const { children, hasError, size = 's' } = props
+  return (
+    <div className={styles.root}>
+      <Text
+        variant='body'
```

Comment:
Are these props mostly defaults? Feels like maybe we don't need a new component for this?

Reply:
Yeah looking at it now, I'm down to delete this as an explicit component. You're right in that there's not much happening in this component and some of it is happening in TextInput anyways (size, for one)